compiler/testData/diagnostics/tests/collectionLiterals/basicCollectionLiterals.fir.kt

```diff
@@ -14,11 +14,11 @@ fun test() {
 }
 
 fun check() {
-    [1, 2] <!INAPPLICABLE_CANDIDATE!>checkType<!> { <!INAPPLICABLE_CANDIDATE!>_<!><Array<Int>>() }
-    [""] <!INAPPLICABLE_CANDIDATE!>checkType<!> { <!INAPPLICABLE_CANDIDATE!>_<!><Array<String>>() }
+    [1, 2] <!UNSAFE_CALL!>checkType<!> { <!INAPPLICABLE_CANDIDATE!>_<!><Array<Int>>() }
```

Comment:
There are many, many cases like this. Somehow `// !CHECK_TYPE` is not quite working well with FIR yet?